internal/params/openvpn.go

```diff
@@ -21,6 +21,7 @@ func (p *paramsReader) GetUser() (s string, err error) {
 	} else if len(s) == 0 {
 		return s, fmt.Errorf("USER environment variable cannot be empty")
 	}
+	s = strings.Replace(s, ' ', '', -1)

```

Comment:
Why is this needed? You got white spaces where? Trailing spaces?

Reply:
Not 100% needed, but a nice to have if you want.

Mullvad includes spaces in the username they give you on their site by default but don't accept spaces connecting through openvpn, so stripping them for convenience if users copy/paste directly from site.

Reply:
Hmm I guess no one has a username with spaces, so good idea.

You can use `strings.ReplaceAll` instead though :wink`

Reply:
Also I think you forgot to import `"strings"` at the top so the thing does not compile.

As mentioned, you might want to try [the development container](https://github.com/qdm12/private-internet-access-docker#development) as it lints warnings/errors and auto-imports everything for you 👍